modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -38,6 +38,9 @@
                   label="apostrophe:login"
                   button-type="submit"
                   :modifiers="['gradient-on-hover', 'block']"
+                  :attrs="{
+                    'style': 'height: 47px'
+                  }"
```

Comment:
unrelated to goal, add login-specific styles right into the component.

Reply:
This should be done in the style block of the component.